cmd/oras/manifest/push.go

```diff
@@ -43,7 +43,7 @@ type pushOptions struct {
 func pushCmd() *cobra.Command {
 	var opts pushOptions
 	cmd := &cobra.Command{
-		Use:   "push [flags] name[:tag|@digest] file",
+		Use:   "push [flags] <name>[:<tag>|@<digest>] <file>",
```

Comment:
I just found a UX inconsistency here.

We should remove the `--media-type` option. Instead, we should
```suggestion
		Use:   "push [flags] <name>[:<tag>|@<digest>] <file>[:<type>]",
```
/cc @qweeah @lizMSFT @yuehaoliang-microsoft @FeynmanZhou